nixos/modules/services/networking/wireguard.nix

```diff
@@ -118,23 +118,52 @@ let
         default = null;
         type = with types; nullOr str;
         example = "container";
-        description = lib.mdDoc ''The pre-existing network namespace in which the
-        WireGuard interface is created, and which retains the socket even if the
-        interface is moved via {option}`interfaceNamespace`. When
-        `null`, the interface is created in the init namespace.
-        See [documentation](https://www.wireguard.com/netns/).
+        description = lib.mdDoc ''
+          The pre-existing network namespace in which the
+          WireGuard interface is created, and which retains the socket even if the
+          interface is moved via {option}`interfaceNamespace`. When
+          `null`, the interface is created in the init namespace.
+          See [documentation](https://www.wireguard.com/netns/).
         '';
       };
 
       interfaceNamespace = mkOption {
         default = null;
         type = with types; nullOr str;
         example = "init";
-        description = lib.mdDoc ''The pre-existing network namespace the WireGuard
-        interface is moved to. The special value `init` means
-        the init namespace. When `null`, the interface is not
-        moved.
-        See [documentation](https://www.wireguard.com/netns/).
+        description = lib.mdDoc ''
+          The pre-existing network namespace the WireGuard
+          interface is moved to. The special value `init` means
+          the init namespace. When `null`, the interface is not
+          moved.
+          See [documentation](https://www.wireguard.com/netns/).
+        '';
+      };
+
+      fwMark = mkOption {
+        default = null;
+        type = with types; nullOr string;
+        example = "0x6e6978";
+        description = lib.mdDoc ''
+          Mark all wireguard packets originating from
+          this interface with the given firewall mark. The firewall mark can be
+          used in firewalls or policy routing to filter the wireguard packets.
+          This can be useful for setup where all traffic goes through the
+          wireguard tunnel, because the wireguard packets need to be routed
+          differently.
+        '';
+      };
+
+      mtu = mkOption {
+        default = null;
+        type = with types; nullOr int;
+        example = "1280";
```

Comment:
```suggestion
        example = 1280;
```

Should be an int.

Reply:
the formatting commit changes documentation of already existing options. if it does not belong here, I can just drop it.

Reply:
Do you forget to change this?